packages/cozy-intent/src/utils/index.ts

```diff
@@ -18,3 +18,15 @@ export const interpolate = (
     return str
   }
 }
+
+declare const __DEVELOPMENT__: boolean | void
+
+export const warn = (message: string): void => {
+  try {
+    if (__DEVELOPMENT__) {
+      // eslint-disable-next-line no-console
+      console.warn(message)
```

Comment:
let's use logger from cozy-logger please 🙏

Reply:
I wanted to keep the 0 dependency constraint here. Would you deem it unwanted? Though cozy-logger is 1.5kb of course, which is not a lot

Reply:
Remember our internal rules: If a cozy lib depends of another cozy-libs, then it's not a direct dep. It's a devDep + peerDep. The consuming app should already have the wanted dep